src/framework/handlers/hierarchy.js

```diff
@@ -10,6 +15,7 @@ class HierarchyHandler {
     handlerType = "hierarchy";
 
     constructor(app) {
+        super(app);
         this._app = app;
```

Comment:
Since `ResourceHandler#constructor` is setting `this._app` now, should we remove it in other places?